src/sentry/similarity/__init__.py

```diff
@@ -57,21 +60,33 @@ def get_frame_attributes(frame):
     return attributes
 
 
-features = FeatureSet(
-    MinHashIndex(
-        redis.clusters.get(
-            getattr(
-                settings,
-                'SENTRY_SIMILARITY_INDEX_REDIS_CLUSTER',
-                'default',
-            ),
-        ),
+def _make_index(cluster=None):
+    if not cluster:
+        cluster_id = getattr(
```

Comment:
I didn't check that this was ready to go (I'm assuming it is and that it is of the correct type.)

Reply:
This is configured over here https://github.com/getsentry/getsentry/pull/1318

Reply:
Agh, not sure how I missed that one, thanks :dizzy_face: